frontend/src/metabase/questions/containers/CollectionEditorForm.jsx

```diff
@@ -41,12 +42,14 @@ export default class CollectionEditorForm extends Component {
                 form
                 title={fields.id.value != null ? fields.name.value : "New collection"}
                 footer={[
-                    <Button className="mr1" onClick={onClose}>
-                        Cancel
-                    </Button>,
-                    <Button primary disabled={invalid} onClick={handleSubmit}>
-                        { fields.id.value != null ? "Update" : "Create" }
-                    </Button>
+                    <div ref={(ref) => { this.footerRef = ref}}>
```

Comment:
Adding this `ref` just for the test seems a little wrong, but I see why you did it. I think ideally we'd be more strict about having the modal content separated from the actual modal itself to make things more easily testable.